README.md

```diff
@@ -128,6 +128,13 @@ If we want our earnings to be in `tokenA` then the
 commodity must be `tokenB`. So we can buy from the sell order, `20 tokenB` using `8 tokenA`, then
 using these `20 tokenB` we can get `10 tokenA` from the buy order, earning `2 tokenA`.
 
+> **ⓘ **
```

Comment:
There is a GitHub specific Markdown extension that allows to place special colored content with icons:
 - https://github.com/orgs/community/discussions/16925

I think we should consider using such a box here. `> [!NOTE]  ` sounds like a good choice.